loader.js

```diff
@@ -34,12 +35,25 @@ module.exports = async function mdxEnhancedLoader(src) {
 
   // Checks if there's a layout, if there is, resolve the layout and wrap the content in it.
   processLayout
-    .call(this, options, data, content, resourcePath)
+    .call(this, options, data, content, resourcePath, scans)
     .then(result => callback(null, result))
     .catch(err => callback(err))
 }
-
-function processLayout(options, frontMatter, content, resourcePath) {
+function scanContent(options, content) {
+  const { mdxEnhancedPluginOptions: pluginOpts } = options
+  if (typeof pluginOpts.scan === 'undefined') return []
```

Comment:
```suggestion
if (!pluginOpts.scan) return []
```

Reply:
Had this before but there was something stale on CI, I changed literally just to trigger a refresh... I will switch back